clang/include/clang/Basic/Attr.td

```diff
@@ -1275,6 +1275,20 @@ def SYCLSimdAccessorPtr : InheritableAttr {
   let Documentation = [Undocumented];
 }
 
+// Used by FE to mark SYCL kernel pointer parameters which correspond to the
+// original lambda's captured accessors. FE turns the attribute to some metadata
```

Comment:
I think you can just say - `Used by SYCL FE to mark kernel parameters which correspond to the data pointer of captured accessors. `